api/database/seeders/BigSeederNotifications.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use App\Notifications\System as SystemNotification;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Database\Seeder;
+
+class BigSeederNotifications extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * This seeds a lot of data
+     * Run this AFTER core data has been seeded, this will not seed platform data
+     *
+     * Notifications
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $input = $this->command->ask('Please enter how many notifications to send every user');
+        $limit = intval($input);
+        $limit = (is_int($limit) && $limit > 0) ? $limit : 1;
+
+        // constant values for reuse and setup
+        $viewGroup = 'notification_test';
+        $notification = new SystemNotification(
+            channelEmail: false,
```

Comment:
This one performed just well for a lot of users ..
instead of https://example.org , if any in app url would be  helpful .. as currently touching this notification triggers firewall error in our verticals

. {
    "href": {
        "en": "https://example.org",
        "fr": "https://example.org"
    },
    "message": {
        "en": "This is a test notification.\n",
        "fr": "Il s'agit d'une notification de test.\n"
    }
}